src/LinearAlgebra/COO_Matrix.hpp

```diff
@@ -679,18 +748,21 @@ inline bool COO_Matrix<ScalarT, Intdx>::checkIncreaseSize(Intdx r, Intdx c)
 }
 
 /**
- * @brief Sort a disoreded set of values_. Assume nothing on order.
+ * @brief Sorts unordered COO matrix
  * 
```

Comment:
Are duplicates allowed?

Reply:
No. They are resolved after every addition.

Reply:
This should be documented immediately after the `@brief` stanza.